backend/src/main/java/de/neuefische/backend/controller/TastyDateController.java

```diff
@@ -15,12 +16,25 @@
 @RequestMapping("/api/appointment")
 public class TastyDateController {
 
-    private final DateSettingsService dateInfoService;
+    private final TastyDateSettingsService dateInfoService;
 
-    public TastyDateController(DateSettingsService dateInfoService) {
+    public TastyDateController(TastyDateSettingsService dateInfoService) {
         this.dateInfoService = dateInfoService;
     }
 
+
+    @GetMapping("/getAllTastyDateItems")
```

Comment:
Try to stick why the naming conventions 👍 https://restfulapi.net/resource-naming/

```suggestion
    @GetMapping
```

Reply:
I checked the hyperlink and altered the names of my REST API